docs/source/cmip_ap.rst

```diff
@@ -0,0 +1,33 @@
+===========================
+NCAR CMIP Analysis Platform
+===========================
+
+`NCAR's CMIP Analysis Platform (CMIP AP) <https://www2.cisl.ucar.edu/resources/cmip-analysis-platform>`_
+inncludes a large collection of CMIP5 and CMIP6 data sets. 
+
+Requesting data sets
+--------------------
+
+`Use this form <https://www2.cisl.ucar.edu/resources/cmip-analysis-platform/request-cmip6-data-sets>`_
+to request new data be added to the CMIP AP. Typically requests are fulfilled
+within two weeks. Contact `CISL <https://www2.cisl.ucar.edu/user-support/getting-help>`_
+if you have further questions. Intake-ESM catalogs are regularly updated
```

Comment:
Note the ambiguity here. Would be useful to have this clarified on the CMIP AP webpage.